.snyk

```diff
@@ -5,14 +5,18 @@ ignore:
     - resources/python/bin/3.7/explicit_requirements.txt
     - resources/python/bin/3.9/explicit_requirements.txt
     - resources/python/bin/3.10/explicit_requirements.txt
+    - resources/python/bin/3.11/explicit_requirements.txt
     - resources/python/src/3.7/explicit_requirements.txt
     - resources/python/src/3.9/explicit_requirements.txt
     - resources/python/src/3.10/explicit_requirements.txt
+    - resources/python/src/3.11/explicit_requirements.txt
```

Comment:
No wildcards possible?

Reply:
Unfortunately nothing in [their documentation](https://docs.snyk.io/manage-risk/policies/the-.snyk-file).